src/rules-runner.ts

```diff
@@ -130,6 +130,19 @@ export class RulesRunner {
     }
   }
 
+  runCustomRegexReplacement(customRegexs: CustomRegex[], oldText: string): string {
+    logDebug(`Running Custom Regex`);
+    let tempOldText = oldText;
+    for (const eachRegex of customRegexs) {
+      if (!eachRegex.find || eachRegex.find.trim() == '' || !eachRegex.replace || eachRegex.replace.trim() == '') {
+        continue;
+      }
+      const regex = new RegExp(`${eachRegex.find}`, 'gm');
```

Comment:
Is there a reason not to allow for the user to select the regex flags? I know with custom ignore, I am allowing them to select which flags to use (I am not going to hold this getting merged in over that as I will likely need to update the display to allow for those selections when I finish up custom ignore).